scripts/openapi/marketo/metadata/metadata.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/amp-labs/connectors/tools/scrapper"
+	"github.com/getkin/kin-openapi/openapi2"
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+func main() {
+	//  read the definitions in the specificatios files.
+	def, docA, err := constructDefinitions("./scripts/openapi/marketo/metadata/asset.json", 5) //nolint:gomnd
+	if err != nil {
+		panic(err)
+	}
+
+	ldef, docL, err := constructDefinitions("./scripts/openapi/marketo/metadata/mapi.json", 4) //nolint:gomnd
```

Comment:
What do the `4` or `5` mean in `constructDefinitions`? Please add a comment for context

Reply:
The `paths` we support  would generate 4 and 5 substrings, when splitted using `/`, for Leads and Assets API respectively.